README.md

```diff
@@ -40,6 +40,12 @@ The library supports Python 3.8-3.10. _Python 3.8 support will be removed soon._
       unset XSIAM_AUTH_ID
       ```
 
+      In case your main git branch is not named **master**, or your upstream is not named **origin**, you can set the following env vars:
+      ```bash
+      export DEMISTO_DEFAULT_BRANCH = main_barnch_title
+      export DEMISTO_DEFAULT_REMOTE = upstream_title
```

Comment:
```suggestion
      In case the primary git branch is not **master**, or the upstream is not named **origin**, set them with environment variables:
      ```bash
      export DEMISTO_DEFAULT_BRANCH = <branch name here>
      export DEMISTO_DEFAULT_REMOTE = <upstream name here>
```